Study Management update for the help guide (#17)
camaradesuk/syrf_userguide_jekyll: whats-new/whats-new-study-management-ui.md

---
title: "25th Jul 2025: Enhanced Study Management Interface"
has_children: false
nav_order: 993
layout: default
parent: What's New!
---

# Updated: Enhanced Study Management Interface

We've unified the management of studies in your SyRF project in the **Study Management** interface. 

> The updated interface now features three clearly organised main sections:

Reviewer: Remove "clearly organised"
```suggestion
The updated interface now features three main sections:
```

> * **Systematic Searches Table:** Your primary workspace where you can upload a new systematic search, bulk update existing studies, start risk of bias calculation, and the actions for viewing, deleting and copying the IDs of studies.

Reviewer: ```suggestion
* **Systematic Searches Table:** Your primary workspace where you can upload a new systematic search, bulk update existing studies including updating screening decisions and adding PDF relative links, initiate automated risk of bias assessment, and view and delete systematic searches from your project.
```

> * **Risk of Bias Assessment Monitoring:** Dedicated section for tracking Risk of Bias assessment processes with detailed progress reporting and comprehensive error reporting.

Reviewer: ```suggestion
* **Risk of Bias Assessment:** Dedicated section for tracking Risk of Bias assessment processes with detailed progress reporting and comprehensive error reporting.
```

> * **Job Progress Monitoring:** Real-time tracking of all background processes including Systematic Search Uploads and Bulk Study Updates, with enhanced error details.

Reviewer: ```suggestion
* **Job Progress Monitoring:** Real-time tracking of all background processes including Systematic Search Uploads and Bulk Study Updates.
```
